src/submission/submission-service.ts

```diff
@@ -18,9 +18,9 @@
  */
 
 import { entities as dictionaryEntities } from '@overturebio-stack/lectern-client';
-import { DataRecord } from '@overturebio-stack/lectern-client/lib/schema-entities';
+import { SchemaValidationError } from '@overturebio-stack/lectern-client/lib/schema-entities';
 import { DeepReadonly } from 'deep-freeze';
-import _ from 'lodash';
+import _, { isEmpty, upperFirst } from 'lodash';
```

Comment:
Can we choose between importing all of lodash or just named methods? Serverside there is no problem with importing all

Reply:
went with default export, as you said no issue serverside plus its mostly used this way in the project